zgrab2_schemas/zgrab2/ipp.py

```diff
@@ -147,15 +147,56 @@
     "request": http_request_full
 })
 
```

Comment:
At some point, it would be nice to give all of the types `doc` tags, and maybe some `example`s too (see e.g. https://github.com/zmap/zgrab2/blob/master/zgrab2_schemas/zgrab2/mysql.py#L62).